pootle/core/debug.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import logging
+import time
+from contextlib import contextmanager
+
+
+logger = logging.getLogger("POOTLE_DEBUG")
```

Comment:
We probably want to consider using `pootle.*` names here to work towards consistency (see #3704 for further background on the matter).